math/examples/riscv_tests/beaglev_fire/sin_performance/hls/config.tcl

```diff
@@ -0,0 +1,17 @@
+source $env(SHLS_ROOT_DIR)/examples/legup.tcl
+set_project PolarFireSoC MPFS025T-FCVG484E Icicle_SoC
+set_parameter POINTSTO_ANALYZE_HW_ONLY 1
+
+#
+# Parameters used for SoC integration
+#
+set_parameter SOC_BD_NAME                 shls_test
```

Comment:
Please a comment that the SOC_BD_NAME *must* match the top_level_name in the gateware .yaml build file.